src/main/java/com/esri/samples/scene/change_atmosphere_effect/README.md

```diff
@@ -0,0 +1,42 @@
+<h1>Change Atmosphere</h1>
+
+<p>Define how the sky looks by changing the visual appearance of the atmosphere in a 3D map (scene).</p>
+
+<p><img src="ChangeAtmosphereEffect.gif" alt="Image" /></p>
+
+<h2>How to use the sample</h2>
+
+<p>Select one of the three available atmosphere effect options. The sky will change to display the selected atmosphere effect. </p>
+
+<h2>How it works</h2>
+
+<p>To change the atmosphere effect:</p>
+
+<ol>
+<li>Create an <code>ArcGISScene</code> and display it in a <code>SceneView</code>.</li>
+
+<li>Change the atmosphere effect with <code>SceneView.setAtmosphereEffect()</code>.</li>
```

Comment:
you can make up a variable to demonstrate the method signature (show what arguments it takes): `sceneView.setAtmosphereEffect(atmosphereEffect)`.